Index: wrappers/pangographite/configure.ac
===================================================================
--- wrappers/pangographite/configure.ac	(revision 724)
+++ wrappers/pangographite/configure.ac	(working copy)
@@ -72,18 +72,6 @@
 # Enable debug
 # ---------------------------------------------------------------------
 if test "$enable_debug" = yes; then
-  if test "$ac_cv_prog_cc_g" = yes; then
-    CFLAGS="$CFLAGS -g -O0 "
-  else
-    CFLAGS="$CFLAGS -O0 "
-  fi
-  if test "$ac_cv_prog_cxx_g" = yes; then
-    CXXFLAGS="$CXXFLAGS -g -O0 "
-  else
-    CXXFLAGS="$CXXFLAGS -O0 "
-  fi
-  CFLAGS="$CFLAGS -Werror -Wall "
-  CXXFLAGS="$CXXFLAGS -Werror -Wall "
   AC_DEFINE(DEBUG)
 else
   AC_DEFINE(NDEBUG)
